spring-cloud-sleuth-core/src/main/java/org/springframework/cloud/sleuth/instrument/zuul/TraceRestClientRibbonCommandFactory.java

```diff
@@ -35,8 +33,16 @@
 import org.springframework.context.ApplicationEventPublisherAware;
 import org.springframework.util.MultiValueMap;
 
+import com.netflix.client.http.HttpRequest;
+import com.netflix.niws.client.http.RestClient;
+
 /**
+ * {@link RestClientRibbonCommandFactory} that ensures that Ribbon requests contain
```

Comment:
Things like this should probably focus on intent.

Propagates traces downstream via http headers that contain trace metadata